internal/create/wizard.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	defaultLength     = 24
 	defaultXKCDLength = 4
+	tplPath           = ".gopass/create/"
```

Comment:
Maybe it would be good to add the related documentation to https://github.com/gopasspw/gopass/blob/master/docs/commands/templates.md ?

Reply:
It's already documented in docs/commands/create.md.